packages/bpk-component-switch/src/BpkSwitch.module.scss

```diff
@@ -43,12 +39,13 @@ $border-width: $bpk-one-pixel-rem * 2;
         @include bpk-themeable-property(
           background,
           --bpk-switch-checked-color,
-          $primary-on-color
+          $bpk-core-accent-day
         );
       }
 
+      // This is now deprecated and will be removed in the future.
       &--event {
-        background: $event-on-color;
+        background: $bpk-color-abisko;
```

Comment:
should we still use this given it's deprecated?

Reply:
As there is no new colour mapping and event type deprecated we have to keep using this. Soon to be removed though once the one usage has been updated